include/mbgl/util/default_styles.hpp

```diff
@@ -0,0 +1,32 @@
+#ifndef MBGL_PLATFORM_DEFAULT_STYLES
+#define MBGL_PLATFORM_DEFAULT_STYLES
+
+#include <vector>
+#include <string>
+
+namespace mbgl {
+namespace util {
+namespace default_styles {
+
+struct DefaultStyle {
+    const char *url;
```

Comment:
Our style for C++ is to space `*` and `&` with the rest of the type, so `const char* url`.